setup.py

```diff
@@ -23,5 +23,13 @@
     ],
     install_requires=[
         'wagtail>=1.0.0',
+        'django-hashers-passlib',
+        'django-storages>=1.4.1',
+        'boto3',
```

Comment:
I don't understand any of this? Why are we requiring all this? The entire world is not using AWS. Nor using contrib comments nor longerusername.

Please think through which packages are actually required and which are optional. You can use [extras](http://setuptools.readthedocs.io/en/latest/setuptools.html#declaring-extras-optional-features-with-their-own-dependencies) in setup.py or just document which packages may be useful.